packages/strapi-plugin-content-type-builder/admin/src/translations/es.json

```diff
@@ -24,29 +80,100 @@
   "form.attribute.item.number.type.decimal": "decimal (ex: 2.22)",
   "form.attribute.item.number.type.float": "flotante (ex: 3.3333333333)",
   "form.attribute.item.number.type.integer": "entero (ej: 10)",
+  "form.attribute.item.privateField": "Campo privado",
+  "form.attribute.item.privateField.description": "Este campo no aparecerpa en la respuesta de la API",
   "form.attribute.item.requiredField": "Campo obligatorio",
   "form.attribute.item.requiredField.description": "No podrá crear un registro si este campo está vacío",
   "form.attribute.item.settings.name": "Ajustes",
+  "form.attribute.item.text.regex": "Patrón RegExp",
+  "form.attribute.item.text.regex.description": "El texto de la expresión regular",
   "form.attribute.item.uniqueField": "Campo único",
   "form.attribute.item.uniqueField.description": "No podrá crear un registro si ya existe otro registro con el mismo contenido",
+  "form.attribute.media.allowed-types": "Seleccionar tipos de multimadia permitidos",
```

Comment:
"Seleccioná los tipos de media que serán permitidos"

Reply:
tipos de media it doesn´t exist. The correct name is "multimedia"